src/io/flutter/preview/PreviewView.java

```diff
@@ -172,16 +173,18 @@ public void loadState(PreviewViewState state) {
   }
 
   public void initToolWindow(@NotNull ToolWindow toolWindow) {
+    sendAnalyticEvent("initToolWindow");
```

Comment:
I think we're already getting this event from listening to general view changes; I think we can avoid sending this specific one in order to match what we do for other view (unless we think this adds some specific value).

Reply:
OK, removing this.